src/backend/app/db/models.py

```diff
@@ -95,7 +95,7 @@ def dump_and_check_model(db_model: BaseModel):
 class DbUserRole(BaseModel):
     """Table user_roles."""
 
-    user_id: int
+    user_id: Optional[int] = None
```

Comment:
Is there a use case for a UserRole entry with no user_id set?

As we won't know the user for which the role relates to